components/SettingsPhone.vue

```diff
@@ -114,26 +106,19 @@ export default {
     },
   },
   watch: {
-    phone(newVal) {
+    phone() {
       if (this.autoSave) {
-        this.savePhone()
+        this.$refs.spinButton.handle()
```

Comment:
I'd prefer it if we restructured `savePhone` so that we could call a method without calling into the spin button.  Partly because it's my preference, but also because I see a lot of issues with use of refs which haven't yet been fully loaded, and it would avoid the risk of that.

Reply:
Oh, yeah, my bad. I've added this for some testing and forgot to remove. SpinButton would never be rendered if autoSave is turned on, so this watch should be left intact. Rolling back that part of the change.